### DISCUSSION TITLE
distance measure

### DISCUSSION THREAD

**Discussion by abliao** [carla-simulator/carla#5313] [Q&A]
I use the sensor to get a picture, then how to get the distance between the cars in the picture and me.

**Comment by LieYuan1993** [comment]
depth camera or get_location() and then calculate the distance

**Reply by abliao** [reply]
I use the sensor to get a picture, then I want to know how to get the distance between the cars in the picture and me.It seems difficult to use get_location() because I don't know the car's Id.I think using depth camera is a good idea, but I want to know whether it can get exactly distance.

**Comment by abliao** [comment]
#5047
I understand how to get the ID from instance segmentation sensor, then I can get the car's location by using get_loacation(), finally I can get the exactly distance.
